_pages/bigquery_schema.md

```diff
@@ -122,6 +122,63 @@ The following fields are deprecated and no longer have meaning in the data set.
 | `string`      |  `Ip_Address`            |
 | `bool`        |  `TruthValue`            |
 
-## Query Examples
+### Blacklist Flags Field
+
+An additional field called ```blacklist_flags``` was added to all tables in response to a "switch discard issue", which identified that some historical tests were affected by traffic microbursts. See our [blog post]({{ site.baseurl }}/blog/traffic-microbursts-and-their-effect-on-internet-measurement) for complete details.
+
+The ```blacklist_flags``` field is used to mark test results if they meet specific parameters that qualify them for inclusion or exclusion in queries or research analyses. This field was created to mark tests affected by the "switch discard issue" identified in 2015-2016, but M-Lab may use the field for other use cases in the future.
+
+Currently, the following values are present in our data in this field:
+
+| Field Name | Value | Description |
+| ---------- | ----- | ----------- |
+| `blacklist_flags` | `0` | unaffected tests |
+| | `1` | affected tests |
+| | `2` | unknown cases |
```

Comment:
Suggest changing "unknown cases" to `tests not shown to be unaffected by switch discards`.  We know something about them: we know is that we can't prove it one way or the other based on available evidence.